Dockerfile.build.tmpl

```diff
@@ -116,16 +111,15 @@ RUN echo "build --spawn_strategy=standalone --genrule_strategy=standalone" \
 
 WORKDIR /
 
-RUN git clone $DEEPSPEECH_REPO
+RUN git clone --recursive $DEEPSPEECH_REPO
 WORKDIR /DeepSpeech
 RUN git checkout $DEEPSPEECH_SHA
-
-# Link DeepSpeech native_client libs to tf folder
-RUN ln -s /DeepSpeech/native_client /tensorflow
+RUN git submodule sync tensorflow/
+RUN git submodule update --init tensorflow/
```

Comment:
If you use `git clone --recurse-submodules` you should not need these steps.

Reply:
> If you use `git clone --recurse-submodules` you should not need these steps.

Not according to my testing

Reply:
Ah, sucks. Then ignore these comments. Just terrible that git requires 3 separate commands for this.